### ISSUE TITLE
Some built-in commands not working on Ubuntu

### ISSUE THREAD

**Issue by ismlkrkmz** [mertyildiran/Dragonfire#10]
When I say the commands that I mentioned below, there is **no error or failure** but at the same time there is **no action at all**.
**PS:** I'm trying all these on Ubuntu 14.04 LTS.

Here are the commands that not worked as its expected: 

```
FILE MANAGER / OPEN FILES
OPEN BLENDER
PHOTOSHOP / PHOTO EDITOR
INKSCAPE
VIDEO EDITOR
OPEN CAMERA
OPEN CALENDAR
OPEN CALCULATOR
OPEN STEAM
```

For packages that are not coming with official Ubuntu distribution like Steam, Blender, Inkscape, Gimp, Kdenlive etc.
**Expected behaviour:**  "[PACKAGE_NAME] Not Installed, Sir!"

 For File Manager / Open Files, Calculator and Calendar commands **related triggers are truly missing**.

**Comment by mertyildiran** [comment]
> **Expected behaviour:** "[PACKAGE_NAME] Not Installed, Sir!"
is actually a good point. :+1: 

@ismlkrkmz Please, find and suggest the proper Shell commands for File Manager / Open Files, Calculator and Calendar commands. So we can make an update.

**Comment by ismlkrkmz** [comment]
@mertyildiran OK, I will.

**Comment by ismlkrkmz** [comment]
@mertyildiran All we need is here:

**File Manager / Open Files:**
`nautilus --browser ~/some/directory` 

**Calculator:**
`gnome-calculator`

**Calendar (Orage):**
`orage`
**NOTE: Orage Calendar should be installed!**

**Comment by mertyildiran** [comment]
Solved with: 9b9a1de46fa544ca98ea7d6981c579d03319c243
